types/stripe/index.d.ts

```diff
@@ -4090,20 +4090,20 @@ declare namespace Stripe {
 
         interface IBankAccountHash extends IResourceObject {
             /**
-             * value is "customer_bank_account"
+             * value is "bank_account"
              */
-            object: "customer_bank_account";
+            object: "bank_account";
 
             /**
              * The name of the person or business that owns the bank account.
              */
-            account_holder_name: string;
+            account_holder_name: string | null;
 
             /**
              * The type of entity that holds the account. This can be either
              * "individual"" or "company".
              */
-            account_holder_type: string;
+            account_holder_type: "individual" | "company" | null;
```

Comment:
These are the only valid values: https://stripe.com/docs/api/customer_bank_accounts/object#customer_bank_account_object-account_holder_type

Stripe will return `null` if the value is not set.